src/checks_linux.go

```diff
@@ -21,6 +21,12 @@ func processCheck(check *check, checkType string, arg1 string, arg2 string, arg3
 		}
 		result, err := guestDisabledLDM()
 		return err == nil && !result
+	case "PasswordChanged":
+		if check.Message == "" {
+			check.Message = "Insecure password for " + arg2 + " has been changed"
```

Comment:
Not necessarily an insecure password, may be more accurate to just say password changed